examples/CMakeLists.txt

```diff
@@ -73,15 +72,19 @@ else()
             "to be turned ON - skipping")
 endif()
 
+# TODO: it looks like there is some problem with IPC implementation in Level
+# Zero on windows
 if(UMF_BUILD_GPU_EXAMPLES
    AND UMF_BUILD_LIBUMF_POOL_DISJOINT
-   AND UMF_BUILD_LEVEL_ZERO_PROVIDER)
+   AND UMF_ENABLE_POOL_TRACKING
```

Comment:
`UMF_ENABLE_POOL_TRACKING` not available anymore

Reply:
Done